pkg/apis/clusterlink.net/v1alpha1/import.go

```diff
@@ -51,9 +53,15 @@ type ImportSpec struct {
 	// TODO: add LoadBalancingSpec.
 }
 
+const (
+	// ImportServiceCreated is a condition type for indicating whether the import service was successfully created.
+	ImportServiceCreated string = "ImportServiceCreated"
```

Comment:
nit: since this is part of the ImportStatus, consider shortening to `ServiceCreated`?

Reply:
This const is part of the `v1alpha1` package, so you need to prefix it with `Import`.

Reply:
I meant on the string value, not variable name (i.e., `ImportServiceCreated string = "ServiceCreated"`).

k8s doesn't seem to do that. For example, [Service Types](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types) don't have a `Service` prefix (i.e., `ClusterIP` not `ServiceClusterIP`).
Just FYI - don't have a preference here.